### ISSUE TITLE
[NEWS] Możesz wystawić maksymalnie 100 000 ofert na jednym koncie

### ISSUE THREAD

**Issue by TomekKaczmarek** [allegro/allegro-api#367]
Wdrożyliśmy dziś limit ilości aktywnych ofert na jednym koncie. **Sprzedawca może wystawić maksymalnie 100 000 ofert**. Zmiana wpływa na następujące metody:

[doNewAuctionExt](https://allegro.pl/webapi/documentation.php/show/id,113)
Sprzedawca nie wystawi oferty ponad limit 100 000 ofert. Jeśli na danym koncie przekroczyłeś limit, otrzymasz komunikat: 
<faultcode>ERR_PUBLICATION_MAX_ACTIVE_OFFERS</faultcode>
<faultstring>Nie można wystawić oferty - Twoje konto przekroczyło maksymalną liczbę aktywnych ofert.</faultstring>
<faultactor>doNewAuctionExt - Offers cannot be published - your account would exceeded the maximum number of active offers</faultactor>

[doSellSomeAgain ](https://allegro.pl/webapi/documentation.php/show/id,321)i [doSellSomeAgainInShop](https://allegro.pl/webapi/documentation.php/show/id,322)
Jeśli sprzedający przekroczył limit ofert na danym koncie, nie wystawimy kolejnej oferty.

**Comment by lmajgier** [comment]
Czy ten temat jest nadal aktualny? Są sprzedawcy którzy mają więcej niż 100,000 produktów na jednym moncie.
My też będziemy tak mieć, więc w jaki sposób mamy je wystawiać, skoro tutaj piszecie o limicie 100k per konto.

**Comment by PrzemyslawLukanowski** [comment]
Tak, limit nadal wynosi 100 000 produktów. Jeżeli czujesz potrzebę, abyśmy znieśli ten limit dla Twojego konta, skontaktuj się ze swoim opiekunem handlowym.

**Comment by lmajgier** [comment]
Jakim opiekunem handlowym? skąd mogę wziąć kontakt do opiekuna handlowego? 
tzn, że muszę być "VIP" aby dodać więcej produktów do sprzedaży?

**Comment by PrzemyslawLukanowski** [comment]
Dokładnie tak.
